ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml

```diff
@@ -417,10 +426,70 @@ spec:
         - name: kerberos_keytab_secret_key
           value: "$(params.kerberos_keytab_secret_key)"
 
+    # send UMB message for RADAS to sign the container image
+    - name: request-signature
+      runAfter:
+        - publish-bundle
+      taskRef:
+        name: request-signature
+      params:
+        - name: pipeline_image
+          value: "$(params.pipeline_image)"
+        - name: manifest_digest
+          value: "$(tasks.publish-bundle.results.manifest_digests)"
+        - name: reference
+          value: "$(tasks.publish-bundle.results.docker_references)"
+        - name: requester
+          value: "amisstea"
```

Comment:
Shouldn't this be an email?

Reply:
If it must be a username, could we use a service account backed by ldap?

Reply:
So is this okay as is? @Allda @amisstea

Reply:
I'm ok with it. I'll also document this in the guild. The guild master should be the one requesting signatures for as long as this is a requirement.